packages/bot-web-ui/src/components/summary/summary-card.types.ts

```diff
@@ -39,14 +52,22 @@ export type TContractInfo = {
     profit: number;
     profit_percentage: number;
     purchase_time: number;
+    sell_price?: number;
+    sell_spot?: number;
+    sell_spot_display_value?: string;
+    sell_spot_time?: number;
+    sell_time?: number;
     shortcode: string;
     status: string;
+    tick_count?: number;
+    tick_stream?: Array<TTickStream>;
     transaction_ids: TTransactionIds;
     underlying: string;
     validation_error: string;
+    validation_error_code?: string;
 };
 
 export interface TSummaryCardProps {
-    contract_info?: TContractInfo | null;
+    contract_info?: TCoreStores['portfolio']['active_positions'][0]['contract_info'] | null;
```

Comment:
```suggestion
    contract_info?: contract_info: ProposalOpenContract &
        Portfolio1 & {
            contract_update?: ContractUpdate;
        };
```
@maryia-matskevich-deriv can we take the types directly from `'@deriv/api-types';` ?

Reply:
@shafin-deriv Yes, we have options, we can use this option

Reply:
✅done